README.md

```diff
@@ -113,7 +113,7 @@ own machine to see for yourself.
 ## Learn more
 **:dart: [Design Goals](#design-goals)** - learn about the design goals of the project (features, performance metrics, and more).
 
-**:seedling: [Documentation](docs/GettingStarted.md#getting-started)** - learn how to reference the parser from your project, and how to perform
+**:book: [Documentation](docs/GettingStarted.md#getting-started)** - learn how to reference the parser from your project, and how to perform
```

Comment:
@jmarolf If only you had a "growth mindset", you would understand the beauty of the 🌱 metaphor.... you _**are**_ the seedling, Jon, but alas... it's come to this. :wink: